include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_fpga.h

```diff
@@ -247,6 +280,17 @@ __parallel_find(_ExecutionPolicy&& __exec, _Iterator __first, _Iterator __last,
     return oneapi::dpl::__par_backend_hetero::__parallel_find(__device_policy, __first, __last, __f, __is_first);
 }
 
+template <typename _ExecutionPolicy, typename _Iterator, typename _Brick, typename _IsFirst>
+_Iterator
+__parallel_find(oneapi::dpl::__internal::__fpga_backend_tag __tag, _ExecutionPolicy&& __exec, _Iterator __first, _Iterator __last, _Brick __f, _IsFirst __is_first)
```

Comment:
Could you please try to remove this overload and make sure the device one is called?

Reply:
Fixed